lib/puppet/ssl/host.rb

```diff
@@ -371,27 +378,61 @@ def state
 
   private
 
+  # Ensures that CRL is either on disk, or that CRL checking has been disabled
+  def ensure_crl!
```

Comment:
Things that might hit the network probably don't need exclamation points, since it seems like nearly any method might eventually trigger a network call.... I don't know if there's a convention in Puppet around ending methods with `!` but I didn't want to highlight that aspect of this method.